cloud-ide.md

```diff
@@ -1 +1,267 @@
+    ```sh
+    cd projects
+    ```
+
+2. We can create a new rails app, install dependencies (done as part of `rails new`), and move into our new project folder using the following commands:
+   ```sh
+   rails new test2
+   cd test2
+   ```
+ 
+### Step 2: Setting up the Database (Postgres)
+
+Next, we need to set up our application to use postgres:
+
+1. Ensure postgres is running:
+    ```sh
+    service postgresql start
+    ```
+2. Open up the `config/database.yml` file in your editor and change it so that it looks exactly like the below: 
+    ```yml
+    default: &default
+      adapter: postgresql
+      encoding: unicode
+      pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+      username: postgres
+      password: password
+      host: localhost
+
+    development:
+      <<: *default
+      database: splurty_development
+
+    test:
+      <<: *default
+      database: splurty_test
+
+    production:
+      <<: *default
+      database: splurty_production
+      # username: splurty
+      # password: <%= ENV['SPLURTY_DATABASE_PASSWORD'] %>
+    ```
+3. Add the `pg` gem and install it for the project by typing in the terminal:
+    ```sh
+    bundle add pg
+    ```
+4. Set up the application database:
+    ```sh
+    rake db:create
+    ```
+
+### Step 3: Setting up the Run Command
+
+Finally, we need to create a new run command so that we can run the application and preview the live URL. We can do this through the UI:
+
+1. Create a new run command by selecting the commands tab or by clicking the play icon in the top right:
+
+   <img width="700" alt="codenvy create run command" src="images/codenvy/project-setup/1-create-command.png" />
+
+2. Enter the following configuration information for the run command you're creating:
+
+   - Name: `rails`
+   - Command Line:
+      ```sh
+      cd ${current.project.path} && rails server -b 0.0.0.0
+      ```
+   - Preview URL: `http://${server.port.3000}`
+  
+     <img width="700" alt="codenvy run command config" src="images/codenvy/project-setup/2-command-info.png" />
+
+3. Click the **Save** button.
+
+4. Run the process by clicking the play button in the top area:
+
+   <img width="700" alt="codenvy run process" src="images/codenvy/project-setup/3-run.png" />
+
+5. Find the live URL in the process pane:
+
+   <img width="700" alt="codenvy find hosted link" src="images/codenvy/project-setup/4-find-link.png" />
+
+6. Click the link to see the resulting application!
+
+   <img width="700" alt="codenvy resulting app" src="images/codenvy/project-setup/5-result.png" />
+
+Congratulations, your application is now up and running!
+
+If you want to stop your running application, simply close the rails process in the terminal/process pane.
+
+> Note: Remember <a href="#workspace-note">this note regarding how your workspace will reset after it has been stopped</a>. Also remember to re-navigate into your project using `cd projects/splurty`.
```

Comment:
This is the instruction to how to use the `splurty`, but I think maybe it should be generic?